### PR TITLE
SONARPY-1945: Added fail fast on stub generation.

### PR DESCRIPTION
**Pull request by joke1196**

### CODE REVIEW

**Comment by joke1196** [comment]
The CI successfully fail when the checksum are different. 
Screenshot from 2024-06-24 13-54-14

File: python-frontend/typeshed_serializer/runners/tox_runner.py [hunk 1]
```diff
@@ -127,8 +127,11 @@ def main(skip_tests=False):
     logger.info(f"Current checksum {current_sources_checksum}")
     logger.info(f"Checksum is computed over {len(source_files)} files")
     if previous_sources_checksum != current_sources_checksum:
-        logger.info("STARTING TYPESHED SERIALIZATION")
-        subprocess.run(["tox"], check=True)
+        if not fail_fast:
+            logger.info("STARTING TYPESHED SERIALIZATION")
+            subprocess.run(["tox"], check=True)
+        else:
+            raise RuntimeError('INCONSISTENT BINARY CHECKSUMS')
```
Comment (ghislainpiot) [inline python-frontend/typeshed_serializer/runners/tox_runner.py:134]: I would have swapped the conditions and the branches to have `if fail_fast: ...`

**Review by ghislainpiot — APPROVED** [review]
LGTM! I have a small comment about the readability of the `if` condition

**Review by ghislainpiot — APPROVED** [review]
Looks good to me!

**Pull request merged**